version.go

```diff
@@ -5,8 +5,8 @@ const (
 	Version = "0.10.9"
 
 	// FluentBitImage is the image for slow-log sidecar container.
-	FluentBitImage = "quay.io/cybozu/fluent-bit:1.7.8.1"
+	FluentBitImage = "quay.io/cybozu/fluent-bit:1.8.11.1"
 
 	// ExporterImage is the image for mysqld_exporter sidecar container.
-	ExporterImage = "quay.io/cybozu/mysqld_exporter:0.13.0.1"
+	ExporterImage = "quay.io/cybozu/mysqld_exporter:0.13.0.3"
```

Comment:
I don't think `mysqld_exporter` needs an update.  It is a statically linked binary on a scratch image.

Reply:
@ymmt2005 
Thank you. I reverted the exporter version.